src/topology/topological_fiber_bundle.lean

```diff
@@ -979,3 +1051,111 @@ begin
 end
 
 end topological_fiber_bundle_core
+
+section topological_fiber_prebundle
+
+variables (F) {Z : Type*} [topological_space B] [topological_space F] {proj : Z → B}
+
+/-- This structure permits to define a fiber bundle when trivializations are given as local
+equivalences but there is not yet a topology on the total space. The total space is hence given a
+topology in such a way that there is a fiber bundle structure for which the local equivalences
+are also local homeomorphism and hence local trivializations. -/
+@[nolint has_inhabited_instance]
+structure topological_fiber_prebundle (proj : Z → B) :=
+(trivialization_at : B → prebundle_trivialization F proj)
+(trivialization_at_mem : ∀ x : B, x ∈ (trivialization_at x).base_set)
+(cont_triv_inter : ∀ x y : B, continuous_on ((trivialization_at x) ∘
+  (trivialization_at y).to_local_equiv.symm) ((trivialization_at y).target ∩
+  ((trivialization_at y).to_local_equiv.symm ⁻¹' (trivialization_at x).source)))
+
+variable {F}
+
+/-- Topology on the total space that will make the prebundle into a bundle. -/
+def topological_fiber_prebundle.total_space_topology (a : topological_fiber_prebundle F proj) :
+  topological_space Z :=
+⨆ x : B, coinduced (a.trivialization_at x).set_symm (subtype.topological_space)
+
+lemma preimage_nhds_within_coinduced {α : Type*} {β : Type*} [topological_space α] {π : α → β}
```

Comment:
This one is also out of place here. Otherwise, LGTM!

Reply:
Same as above, sorry...

Reply:
Just because I needed it like that but I can change it back

Reply:
Since you are in an open set, this should be equivalent. and having a full neighborhood is always easier to use. So yes, I think it is a good change.